static/js/vue-cdr-access/src/mixins/routeUtils.js

```diff
@@ -125,9 +125,13 @@ export default {
         /**
          * Returns a list of query parameters from the current URLs minus the specified set of parameters
          * @param param_names names of query parameters to remove
+         * @param reset_start_row Start row should be reset to 0 for searching, pagination purposes
          * @returns {LocationQuery} list of query parameters with specified parameters removed
          */
-        removeQueryParameters(param_names) {
+        removeQueryParameters(param_names, reset_start_row = false) {
+            if (reset_start_row) {
```

Comment:
I think I would prefer to add a new method named "resetStartRow" which we could used like 
```
this.routeWithParams(this.resetStartRow(this.removeQueryParameters(this.allPossibleSearchParameters)))
```
It'd be clearer what is happening and it wouldn't add extra behaviors into this method, which is pretty clearly scoped right now